atomic

```diff
@@ -683,6 +682,12 @@ def create_parser(help_text):
     verifyp.add_argument("-v", "--verbose", default=False,
                           action="store_true",
                           help=_("Report status of each layer"))
+    verifyp.add_argument("-V", "--validate", default=True,
+                                action="store_false",
+                                help=_("disable validating system images"))
+    verifyp.add_argument("-g", "--generate", default=False,
```

Comment:
That sort of thing is why I'm not using the immutable bit in ostree today.  The semantics are annoying.  My proposal for O_OBJECT *does* allow root to unlink the file (and to create new links to it).  I.e. the inode can change, but not the contents.